src/modules/drawing/lib/drawingUtils.js

```diff
@@ -154,3 +154,43 @@ export function extractOpenLayersFeatureCoordinates(feature) {
     }
     return coordinates
 }
+
+/**
+ * Checks if point is at target within tolerance.
+ *
+ * @param {Number[]} point The point in question.
+ * @param {Number[]} target The target to check against.
+ * @param {Number} [tolerance=0] Distance from target that still counts. Default is `0`
+ * @returns {Boolean} If the point is close enough to the target.
+ */
+export function pointWithinTolerance(point, target, tolerance = 0) {
+    return (
```

Comment:
Maybe add some defensive measure to make sure `point` is defined 😉

Reply:
As in throwing an error? And for target as well?

Reply:
more something like
```
if (!point || !target) {
    return false
}
```